notebooks/official/custom/custom_training_container_and_model_registry.ipynb

```diff
@@ -749,107 +720,316 @@
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "name_container:training"
+        "id": "a2ed53b6e259"
       },
       "source": [
-        "#### Build the container locally\n",
+        "### Enable Artifact Registry API\n",
         "\n",
-        "Next, you will provide a name for your customer container that you will use when you submit it to the Google Container Registry."
+        "First, you must enable the Artifact Registry API service for your project.\n",
+        "\n",
+        "Learn more about [Enabling service](https://cloud.google.com/artifact-registry/docs/enable-service)."
```

Comment:
To improve parsing, change to:

- Learn more about [enabling the Artifact Registry service](https://cloud.google.com/artifact-registry/docs/enable-service).

Reply:
Did required changes